rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/studio/storage/FileStorageSupportFactory.java

```diff
@@ -70,7 +70,7 @@ private static FileStorageSupport createFromConfig() {
         return new FileStorageSupport(new File(userWorkflowsDir), new File(templateWorkflowsDir));
     }
 
-    private static String relativeToHomeIfNotAbsolute(String path) {
+    public static String relativeToHomeIfNotAbsolute(String path) {
         if (!new File(path).isAbsolute()) {
             path = getSchedulerHome() + "/" + path;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this hard-coded path-delimiter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)